build.gradle

```diff
@@ -251,7 +271,7 @@ subprojects {
   /* JAVADOC ENFORCEMENT */
   // Fail build on javadoc warnings
   tasks.withType(Javadoc) {
-    options.addBooleanOption('Xwerror', true)
+//    options.addBooleanOption('Xwerror', true)
```

Comment:
Note: this causes hundreds of warnings (+ error) of missing javadocs. I can fix, but it will make this PR very large. I suggest I enable as part of a separate PR.

Reply:
yeah you can ignore javadoc for now